controllers/UserMentionsController.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file Controller RESTful Web service API for messages resource

```

Comment:
The docs should reflect that this file relates to UserMentions

Reply:
Sorry about that.. didn't refactor... was going to but fell sick